main.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+from WonderPosition import *
+from WonderWoerter import *
+from WonderBlumen import *
+from WonderUmgebung import *
+from WonderBewegung import *
+from WonderTexte import *
+from WonderInventar import *
+
+
+class Spiel(object):
+    def __init__(self):
+        self.wortVergleicher = WortVergleicher()
+        self.wondertext = WonderText()
+        self.wonderInventar = WonderInventar(self.wortVergleicher)
+        self.umgebungen = UmgebungsGenerator(self.wortVergleicher)      
+        self.bewegung = Bewegung(self.umgebungen.gebeStartUmgebung(),self.wondertext, self.wortVergleicher)
+        self.umgebungen.gebeStartUmgebung().setzeBewegung(self.bewegung)
+        self.DoktormutterPosition = self.umgebungen.gebeDoktormutterPosition()
```

Comment:
Variablen werden nach Konvention klein geschrieben; also 
```suggestion
        self.doktormutterPosition = self.umgebungen.gebeDoktormutterPosition()
```
(ich vermute, das ist hier eher aus Versehen, weil die meisten Anderen sich ja daran halten.)